polarity: 1     # {-1 , 1}
speed : 0.35       # maxturnspeed >=0
yellowZone : 60   # ±angle hysteresis before moving
greenZone : 40     # ±angle tolerance reaching target
